redpanda/examples/python/kafka-produce-avro.py

```diff
@@ -90,31 +116,61 @@ def wrong_form(value_arg):
           " is not of the form " + value_arg_form + ".", file=sys.stderr)
     sys.exit(1)
 
+def is_array(ptype: str):
+    return len(ptype) > 2 and ptype[0:2] == "[]"
+
+def normalize_type(ptype: str):
+    if is_array(ptype):
+        ptype = ptype[2:]
+        prefix = "[]"
+    else:
+        prefix = ""
+    if ptype == "timestamp":
+        return ptype
+    if ptype == "str" or ptype == "string":
+        return prefix + "str"
+    if ptype == "bool" or ptype == "boolean":
+        return prefix + "bool"
+    if ptype == "double" or ptype == "float":
+        return prefix + "float"
+    if ptype == "long" or ptype == "short":
+        return prefix + "int"
+    raise "Unhandled type"
+
+def value_from_str(ptype: str, svalue: str):
+    # Strictly speaking we are calling for a python type here (eg, "str", "int", "float", "bool").
+    # We allow other type names for ease of use for us, people accostumed to Java.
+    if (ptype == "str"):
+        return svalue
+    if (ptype == "bool"):
+        return (svalue == "true" or svalue == "True")
+    if ptype == "double":
+        ptype = "float"
+    elif ptype == "long" or ptype == "short":
+        ptype = "int"
```

Comment:
I don't understand the value of these cases, since `ptype` has already been normalized.
```suggestion
```